cg/services/order_validation_service/workflows/balsamic/rules/case_sample/utils.py

```diff
@@ -0,0 +1,5 @@
+from cg.services.order_validation_service.workflows.balsamic.models.sample import BalsamicSample
+
+
+def has_sex_and_subject(sample: BalsamicSample) -> bool:
+    return sample.subject_id and sample.sex
```

Comment:
I don't think this check works. The Atlas page states that if the sample has a subject_id and a _known_ gender - but sample.sex is an Enum which has a value "unknown". With this check, samples with unknown sex would still be picked up and validated.